tests/functional/adapter/materialized_view_tests/_files.py

```diff
@@ -36,7 +36,8 @@
     cluster_by=["id", "value"],
     enable_refresh=True,
     refresh_interval_minutes=60,
-    max_staleness="INTERVAL 45 MINUTE"
+    max_staleness="INTERVAL 45 MINUTE",
+    allow_non_incremental_definition=True
```

Comment:
It makes sense to add this here. But let's add a check for it as well [here](https://github.com/dbt-labs/dbt-bigquery/blob/063baeb62e59210ad1c278f66c752c510bebcb9b/tests/functional/adapter/materialized_view_tests/test_materialized_view_changes.py#L16-L25). It doesn't test the change, we're doing that with other settings, but it at least tests that we're setting it when it's available.